lib/index.js

```diff
@@ -326,3 +327,65 @@ exports.createStorageClient = async (ctx, accessToken, storageIdPrefix) => {
 
     return storageClient;
 };
+
+exports.createFusebitFunctionFromExpress = (app, { disableStorageClient } = {}) => {
+    // See https://github.com/fusebit/samples/blob/master/express/index.js#L6
+    Object.setPrototypeOf(Object.getPrototypeOf(Object.getPrototypeOf(app.response)), Mock.Response.prototype);
+    Object.setPrototypeOf(Object.getPrototypeOf(Object.getPrototypeOf(app.request)), Mock.Request.prototype);
+
+    return async (ctx) => {
+        exports.debug('HTTP REQUEST', ctx.method, ctx.url, ctx.headers, ctx.body);
+
+        if (!disableStorageClient) {
+            ctx.storage = await exports.createStorageClient(
+                ctx,
+                ctx.fusebit.functionAccessToken,
+                `boundary/${ctx.boundaryId}/function/${ctx.functionId}/root`
+            );
+        }
+
+        let url = ctx.url.split('/');
```

Comment:
Given the various parameters added to ctx, is this splicing magic still necessary?

Reply:
No. Done.